components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/DefaultAdditionalComponentsUIConfigProvider.scala

```diff
@@ -0,0 +1,8 @@
+package pl.touk.nussknacker.engine.api.component
+
+class DefaultAdditionalComponentsUIConfigProvider extends AdditionalComponentsUIConfigProvider {
```

Comment:
Let's call it `Empty...`. I'd rather expect Map-based implementation as some Default. Or maybe we could have this Map-based implementation and have a `AdditionalComponentsUIConfigProvider.empty` method that would created this class with empty map in the input parameter?